cookbooks/bcpc-hadoop/attributes/monitoring.rb

```diff
@@ -0,0 +1,5 @@
+default["bcpc"]["hadoop"]["zabbix"]["history_days"] = 1
+default["bcpc"]["hadoop"]["zabbix"]["trend_days"] = 15
+default["bcpc"]["hadoop"]["zabbix"]["cron_check_time"] = 240
```

Comment:
No code was changed related to `cron_check`. The attributes were just moved from `default.rb`